cmd/collectors/zapi/plugins/aggregate/aggregate.go

```diff
@@ -52,18 +52,47 @@ func (a *Aggregate) Run(dataMap map[string]*matrix.Matrix) ([]*matrix.Matrix, er
 	if err := a.getCloudStores(); err != nil {
 		if errors.Is(err, errs.ErrNoInstance) {
 			a.Logger.Debug().Err(err).Msg("Failed to collect cloud store data")
-			return nil, nil
 		}
-		return nil, err
+	}
+
+	aggrFootprintMap, err := a.getAggrFootprint()
+	if err != nil {
+		a.Logger.Error().Err(err).Msg("Failed to update footprint data")
+		// clean the map in case of the error
+		aggrFootprintMap = make(map[string]map[string]string)
```

Comment:
`clear(aggrFootprintMap)` unless you need a new instance

Reply:
Done.